Update GUI5.py added missing takeoff to getDroneAction

Arndt-Madison opened the pull request:
added the missing action and added spaces after each if statement to make it easier to read

3C-SCSU reviewed (commented):
**_Code approval process -- 3 Steps review:_**
(1) Yes; (2) Maybe; (3) No; > {Number of files, and total lines changed:add/delete lines} [Reasoning]
**Example:** 
- Feedback >> (2) > {4: +153/-20} [ Code needs to explain what it does ]

3C-SCSU reviewed (changes requested):
- Feedback >> (2) > {1: +21/-0} [ Code needs to explain what it does. For more information, read the comments. ]

Arndt-Madison commented:
while i don't know all the functions that call it and i didn't make it i know from a base overview it is a command safety and execute function. basically it will only execute command it recognizes and handle the error if its not a valid command.
since this is the case i know the action btn (executeAction GUI5:457) calls it and the run command btn (keepDroneAlive GUI5:475) calls it since it existed, there was no point making another or duplicating the code in those functions.

elif action == 'takeoff':
                    self.tello.takeoff()
                    self.logMessage.emit("Taking off")
                    self.flight_log.insert(0, "Taking off")
                    
the code above is what i added since it was missing takeoff. just from looking at it the elif is looking for that spesific command, the tello line is executing the command and the last 2 appear to be a log.
since i copied and pasted one of the other filters and modified it for the takeoff command and testing it. what i do know is it calls tello and places a action in the flight log on the GUI

Arndt-Madison commented:
> * Feedback >> (2) > {1: +21/-0} [ Code needs to explain what it does. For more information, read the comments. ]

this is not a function i made, i only added a else if and separated the if statements to make it readable. i will add comments for the if statements as i know for serenity what they do. however i dint make the code so i cant guaranty comments on the rest will be accurate. 

before i make the changes is that alright with you??

Arndt-Madison commented on GUI5.py:
```diff
@@ -516,77 +516,98 @@ def record_action(name, value=None):
+                    self.tello.takeoff()
+                    self.logMessage.emit("Taking off")
+                    self.flight_log.insert(0, "Taking off")
+				
```
this elif statement did not exist before and is why tonight's takeoff test did not work

3C-SCSU reviewed (approved):
Feedback >> (1) > {1: +21/-0} [ Code solves ticket #636. Well explained and documented. Merging approved. ]

3C-SCSU commented:
Ticket solved: https://github.com/3C-SCSU/Avatar/issues/636
